site/en/posts/bluetooth-microphone-super-resolution-better-nbs-sound-quality-through-machine-learning.md

```diff
@@ -0,0 +1,110 @@
+
+The final processing steps were:
+
+1.  **Augmentation**: The input audio signal was augmented with random cropping, gain adjustment, band-pass filtering, and Gaussian noise.
+1.  **Downsampling**: Audio was down-sampled to 8 kHz; the sampling rate of narrow-band speech.
+1.  **Encoding and decoding**: The CVSD codec was added to make the audio signal distribution similar to that from narrow-band speech connection.
+1.  **Upsampling**: Finally, the audio signal was upsampled to the target sampling rate before feeding into the model. Note that the upsampling here does not reconstruct high-frequency components; instead, it fills the high-frequency bins with aliases of low-frequency bins or simply with zeros.
+
+We also collected audio samples from different Bluetooth microphones for model evaluation, selecting models based on the evaluation results.
+
+## Results preferred by over 80% of users
+
+We presented users with two versions of an audio clip recorded on Airpods: one before processing and one after processing. Our model was able to reconstruct high frequency components generating a sound quality preferred by 82.8% of 450 users.
+
+Listen to the below before and after demo samples processed by the model.
+
+<table>
+  <thead>
+    <tr>
+      <th>Audio signal from Bluetooth NBS connections</th>
+      <th>Audio signal processed by the model</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>
+        <figure>
+            <a href="https://firebasestorage.googleapis.com/v0/b/cros-staging.appspot.com/o/posts%2Fbluetooth-nbs%2Fnbs_input_4_airpods_pro.wav?alt=media">nbs_input_4_airpods_pro.wav</a>
+            <img src="insert_image_url_here" alt="A spectrogram representing the unprocessed audio clip. The scale measures from 0 to 12 kHz, with heat colors presenting a spectrum from -80 dB to +0 dB. The spectrogram shows that the audio clip does not exceed 4 kHz.">
+            <figcaption>From the spectrogram, we can see that the highest frequency is capped at 4 kHz.</figcaption>
+        </figure>
+        </td>
+        <td>
+        <figure>
+            <a href="https://firebasestorage.googleapis.com/v0/b/cros-staging.appspot.com/o/posts%2Fbluetooth-nbs%2Fnbs_processed_4_airpods_pro.wav?alt=media">nbs_processed_airpods_pro.wav</a>
+        <img src="insert_image_url_here" alt="A spectrogram representing the processed audio clip. The scale measures from 0 to 12 kHz, with heat colors presenting a spectrum from -80 dB to +0 dB. The spectrogram shows that the audio clip now peaks up to 12 kHz.">
+        <figcaption>The highest frequency becomes 12 kHz and some of the higher frequency components are reconstructed.</figcaption>
+        </figure>
+    </td>
+    </tr>
+  </tbody>
+</table>
```

Comment:
We don't want to do this as a table. Please make these using our standard table Markdown.